luigi/worker.py

```diff
@@ -412,7 +478,7 @@ def _add(self, task, is_complete):
         elif task.run == NotImplemented:
             deps = None
             status = PENDING
-            runnable = False
+            runnable = configuration.get_config().getboolean('core', 'retry-external-tasks', False)
```

Comment:
This line means that the task will be given to a worker to run (depending on a config value).